repo/packages/K/kibana/800/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "packagingVersion": "4.0",
+  "upgradesFrom": [
+    "2.7.0-6.8.1"
+  ],
+  "downgradesTo": [
+    "2.7.0-6.8.1"
+  ],
+  "minDcosReleaseVersion": "1.11",
+  "name": "kibana",
+  "version": "3.0.0-7.3.2",
+  "maintainer": "support@mesosphere.io",
+  "description": "Kibana 6, and X-Pack",
```

Comment:
```suggestion
  "description": "Kibana 7, and X-Pack",
```

Reply:
Great catch Sam. @phrynchanka let's get this fixed in the Elastic repository as well.

Reply:
Already done. Thanks @mpereira and @samvantran !